SPECS/tdnf/tdnf.spec

```diff
@@ -90,6 +108,7 @@ python bindings for tdnf
 
 %package        autoupdate
 Summary:        systemd services for periodic automatic update
+Group:          Applciations/RPM
```

Comment:
typo: Applications

Reply:
```suggestion
Group:          Applications/RPM
```

Reply:
We don't use these tags, might as well just remove it if we're touching it at all.